go/vt/mysqlctl/s3backupstorage/s3.go

```diff
@@ -166,7 +176,12 @@ func (bh *S3BackupHandle) AddFile(ctx context.Context, filename string, filesize
 		return nil, fmt.Errorf("AddFile cannot be called on read-only backup")
 	}
 
-	partSizeBytes := calculateUploadPartSize(filesize)
+	partSizeBytes, err := calculateUploadPartSize(filesize)
+	if err != nil {
+		return nil, err
+	}
+
+	bh.bs.params.Logger.Infof("Using S3 upload part size: %s", humanize.IBytes(uint64(partSizeBytes)))
```

Comment:
I wonder if it's worth adding a project dependency on humanize for this and the error message below. The flag is in bytes, so I think it would actually be better to log the bytes here as well to confirm that the expected/flag value is used.

Reply:
it was already an indirect dependency, so I thought I would use it :)

I can remove it tomorrow since the preference is to avoid it

Reply:
Ah, you're right! Sorry, for some reason I thought we added it to go.mod here. No reason not to use it here then.

Reply:
okay, I will leave it then and just update the comment as mentioned below!